linkup-cli/src/services/caddy.rs

```diff
@@ -97,3 +136,20 @@ fn write_caddyfile(domains: &[String]) -> Result<()> {
 
     Ok(())
 }
+
+fn check_redis_installed() -> Result<()> {
+    let output = Command::new("caddy")
+        .arg("list-modules")
+        .output()
+        .map_err(|err| CliError::StartCaddy(err.to_string()))?;
+
+    let output_str = String::from_utf8_lossy(&output.stdout);
```

Comment:
**_question:_**
Do you need it to be lossy here? Not sure what is dumped by caddy, but maybe simple from utf8 with bubbling the error work as well?
```rust
let output_str = String::from_utf8(&output.stdout)?;
```

Reply:
👍 , fixed it, it was the gpt suggestion 😅